test/connector/http/basicauth/docker-compose.yml

```diff
@@ -15,14 +15,14 @@ services:
 
   secretless:
     build:
-      context: ../..
+      context: ../../../..
```

Comment:
Let's make sure to add a ZH issue for improving this per our earlier discussion.  If you've already done so, just ignore this comment.